Main.java

```diff
@@ -0,0 +1,73 @@
+package com.company;
+
+import java.io.*;
+import java.util.StringTokenizer;
+
+public class Main {
+
+    public static void main(String[] args){
+
+        if(args.length!=2)
+        {
+            System.out.println("File path and search word cannot be empty");
+            System.exit(0);
+        }
+        String path=args[0];
+        String search=args[1];
+        File file=new File(path);
+        System.out.println("Processing...");
+        doesFileExist(file);
+        wordCount(file,search);
+    }
+
+    private static void doesFileExist(File file)
+    {
+        if (!file.isFile())
+        {
+            System.out.println("File does not exist");
+        } else isFileFormatSupported(file);

```

Comment:
doesFileExist() method should only be concerned with the existence of the file. It should not be calling isFileFormatSupported() from within. Either this second method should be called externally, or you have to rename doesFileExist to validateFile() or something with a similar meaning

Reply:
Also, it would be better to use braces even if its a single line statement.

Reply:
Sure. I will keep that in mind.

Reply:
@premsrj I have  changed it to validateFile() and made all the changes per you suggested.